src/components/EmailAlertsForm/EmailAlertsForm.tsx

```diff
@@ -48,6 +49,8 @@ const EmailAlertsForm: React.FC<{
   const [showConfirmation, setShowConfirmation] = useState(false);
   const formRef = createRef<HTMLFormElement>();
 
+  const isSmallMobile = useBreakpoint(321);
```

Comment:
the text and button start overlapping around 345px. Want to bump this breakpoint up to 360?